models/comments.models.js

```diff
@@ -14,3 +15,22 @@ exports.fetchComments = (articleId) => {
       }
     });
 };
+
+exports.insertComment = (comment, articleID) => {
+  // return checkUserExists(comment.username).then(() => {
+  const { username, body } = comment;
+  if (comment.hasOwnProperty('username') && comment.hasOwnProperty('body')) {
+    return checkUserExists(comment.username).then(() => {
```

Comment:
Not a problem per say but this could also be done with a promise.all to reduce nesting of promises